tests/test_alb.py

```diff
@@ -0,0 +1,166 @@
+from stacker.context import Context
```

Comment:
Can you convert this to the new YAML style tests?  It's actually a lot easier to write, and can serve as really good examples.  See https://github.com/cloudtools/stacker_blueprints/blob/master/tests/test_ecs.yaml for an example.